src/model/footsteps/hooks/index.ts

```diff
@@ -48,16 +52,14 @@ export const useFootsteps = () => {
   // 特定の間隔で足音を再生する関数
   const playSounds = useCallback((speed: number, speedModifier = 1.0) => {
     if (speed < 0) return;
-
-    // 初回の足音を再生
-    const footstepsAudio = loadFootstepsAudio();
-    playAudio(footstepsAudio);
```

Comment:
It's necessary to keep the initial audio playback, as removing it means the audio will only play after the interval has elapsed